2020-12-17 00:34:58.867  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:35:02.108  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:35:02.310  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:35:02.317  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:35:07.324  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:35:07.564  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 201ms. Found 9 JPA repository interfaces.
2020-12-17 00:35:08.524  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:35:08.638  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:35:10.369  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:35:10.413  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:35:10.414  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:35:11.075  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:35:11.076  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 8556 ms
2020-12-17 00:35:11.807  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:35:11.905  INFO 11896 --- [restartedMain] org.hibernate.Version                    : HHH000412: Hibernate ORM core version 5.4.22.Final
2020-12-17 00:35:12.311  INFO 11896 --- [restartedMain] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
2020-12-17 00:35:12.529  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
2020-12-17 00:35:15.017  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Start completed.
2020-12-17 00:35:15.193  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:35:19.100  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:35:19.127  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:35:19.173  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:35:19.398  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:35:20.980  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:35:20.983  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:35:21.018  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:35:21.031  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:35:21.031  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:35:21.047  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:35:21.061  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:35:22.544  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:35:22.920  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 28.746 seconds (JVM running for 31.278)
2020-12-17 00:35:24.479  INFO 11896 --- [http-nio-5000-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:35:24.481  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:35:24.517  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 20 ms
2020-12-17 00:35:24.841  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:35:24.864  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:35:50.809  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : Received MessageCallback from user: gbcD9ezHUeQkbrYUwyU3Bw==, message type: TEXT
2020-12-17 00:35:53.075  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : File message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:35:53.248  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : Text message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:36:18.663  INFO 11896 --- [Thread-6] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:36:18.666  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:36:21.665  WARN 11896 --- [Thread-6] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:36:21.669  INFO 11896 --- [Thread-6] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:36:21.686  INFO 11896 --- [Thread-6] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown initiated...
2020-12-17 00:36:21.727  INFO 11896 --- [Thread-6] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Shutdown completed.
2020-12-17 00:36:22.642  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:36:24.652  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:36:24.653  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:36:24.668  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:36:24.987  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:36:25.057  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 68ms. Found 9 JPA repository interfaces.
2020-12-17 00:36:25.229  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:36:25.242  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:36:25.457  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:36:25.459  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:36:25.460  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:36:25.548  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:36:25.548  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 875 ms
2020-12-17 00:36:25.644  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:36:25.653  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-2 - Starting...
2020-12-17 00:36:25.701  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-2 - Start completed.
2020-12-17 00:36:25.712  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:36:26.516  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:36:26.525  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:36:26.572  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:36:26.737  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:36:27.187  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:36:27.188  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:36:27.197  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:36:27.210  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:36:27.213  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:36:27.826  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:36:28.161  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 6.259 seconds (JVM running for 96.52)
2020-12-17 00:36:28.165  INFO 11896 --- [restartedMain] .ConditionEvaluationDeltaLoggingListener : Condition evaluation unchanged
2020-12-17 00:36:28.973  INFO 11896 --- [http-nio-5000-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:36:28.975  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:36:28.982  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 6 ms
2020-12-17 00:36:29.136  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:36:29.142  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:36:31.805  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : Received MessageCallback from user: gbcD9ezHUeQkbrYUwyU3Bw==, message type: TEXT
2020-12-17 00:36:32.399  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : File message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:36:32.564  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : Text message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:37:18.716 ERROR 11896 --- [threadPoolTaskExecutor-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void by.testbot.services.async.BroadcastServiceAsync.runAsync()

org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:448) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:574) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:361) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at by.testbot.services.PostponeMessageService$$EnhancerBySpringCGLIB$$48612ad8.getAllWithoutLast(<generated>) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.idleMessagesAsync(BroadcastServiceAsync.java:57) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.runAsync(BroadcastServiceAsync.java:49) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync$$FastClassBySpringCGLIB$$1ab3ef16.invoke(<generated>) ~[classes/:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:771) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:509) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:588) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:156) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:466) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:391) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	... 21 common frames omitted

2020-12-17 00:37:41.582  INFO 11896 --- [Thread-11] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:37:41.588  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:37:44.597  WARN 11896 --- [Thread-11] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:37:44.612  INFO 11896 --- [Thread-11] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:37:44.613  INFO 11896 --- [Thread-11] com.zaxxer.hikari.HikariDataSource       : HikariPool-2 - Shutdown initiated...
2020-12-17 00:37:44.629  INFO 11896 --- [Thread-11] com.zaxxer.hikari.HikariDataSource       : HikariPool-2 - Shutdown completed.
2020-12-17 00:37:48.786  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:37:50.851  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:37:50.852  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:37:50.858  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:37:51.765  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:37:51.873  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 106ms. Found 9 JPA repository interfaces.
2020-12-17 00:37:52.090  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:37:52.112  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:37:52.721  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:37:52.723  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:37:52.724  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:37:52.910  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:37:52.910  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 2047 ms
2020-12-17 00:37:53.004  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:37:53.017  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-3 - Starting...
2020-12-17 00:37:53.040  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-3 - Start completed.
2020-12-17 00:37:53.041  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:37:53.767  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:37:53.768  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:37:53.779  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:37:53.845  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:37:54.318  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:37:54.319  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:37:54.327  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:37:54.328  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:37:54.329  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:37:54.857  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:37:55.187  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 10.443 seconds (JVM running for 183.545)
2020-12-17 00:37:55.190  INFO 11896 --- [restartedMain] .ConditionEvaluationDeltaLoggingListener : Condition evaluation unchanged
2020-12-17 00:37:56.047  INFO 11896 --- [http-nio-5000-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:37:56.064  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:37:56.079  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 7 ms
2020-12-17 00:37:56.235  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:37:56.244  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:38:29.724  INFO 11896 --- [Thread-19] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:38:29.725  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:38:32.726  WARN 11896 --- [Thread-19] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:38:32.730  INFO 11896 --- [Thread-19] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:38:32.754  INFO 11896 --- [Thread-19] com.zaxxer.hikari.HikariDataSource       : HikariPool-3 - Shutdown initiated...
2020-12-17 00:38:32.771  INFO 11896 --- [Thread-19] com.zaxxer.hikari.HikariDataSource       : HikariPool-3 - Shutdown completed.
2020-12-17 00:38:33.926  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:38:35.940  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:38:35.941  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:38:35.942  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:38:36.338  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:38:36.406  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 66ms. Found 9 JPA repository interfaces.
2020-12-17 00:38:36.510  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:38:36.522  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:38:36.726  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:38:36.729  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:38:36.756  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:38:36.826  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:38:36.828  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 880 ms
2020-12-17 00:38:36.913  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:38:36.921  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Starting...
2020-12-17 00:38:36.941  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Start completed.
2020-12-17 00:38:36.947  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:38:37.637  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:38:37.638  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:38:37.650  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:38:37.742  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:38:38.084  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:38:38.085  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:38:38.092  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:38:38.103  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:38:38.103  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:38:38.613  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:38:38.965  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 6.04 seconds (JVM running for 227.323)
2020-12-17 00:38:38.967  INFO 11896 --- [restartedMain] .ConditionEvaluationDeltaLoggingListener : Condition evaluation unchanged
2020-12-17 00:38:39.686  INFO 11896 --- [http-nio-5000-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:38:39.688  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:38:39.693  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 5 ms
2020-12-17 00:38:39.846  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:38:39.850  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:38:41.595 ERROR 11896 --- [threadPoolTaskExecutor-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void by.testbot.services.async.BroadcastServiceAsync.runAsync()

org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:448) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:574) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:361) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at by.testbot.services.PostponeMessageService$$EnhancerBySpringCGLIB$$48612ad8.getAllWithoutLast(<generated>) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.idleMessagesAsync(BroadcastServiceAsync.java:57) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.runAsync(BroadcastServiceAsync.java:49) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync$$FastClassBySpringCGLIB$$1ab3ef16.invoke(<generated>) ~[classes/:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:771) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:509) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:588) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:156) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:466) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:391) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	... 21 common frames omitted

2020-12-17 00:38:43.729  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : Received MessageCallback from user: gbcD9ezHUeQkbrYUwyU3Bw==, message type: TEXT
2020-12-17 00:38:44.000  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : File message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:38:44.164  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : Text message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:39:29.731 ERROR 11896 --- [threadPoolTaskExecutor-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void by.testbot.services.async.BroadcastServiceAsync.runAsync()

org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:448) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:574) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:361) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at by.testbot.services.PostponeMessageService$$EnhancerBySpringCGLIB$$48612ad8.getAllWithoutLast(<generated>) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.idleMessagesAsync(BroadcastServiceAsync.java:57) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.runAsync(BroadcastServiceAsync.java:49) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync$$FastClassBySpringCGLIB$$1ab3ef16.invoke(<generated>) ~[classes/:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:771) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:509) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:588) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:156) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:466) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:391) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	... 21 common frames omitted

2020-12-17 00:43:26.554  INFO 11896 --- [Thread-27] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:43:26.555  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:43:29.567  WARN 11896 --- [Thread-27] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:43:29.728  INFO 11896 --- [Thread-27] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:43:29.968  INFO 11896 --- [Thread-27] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Shutdown initiated...
2020-12-17 00:43:30.086  INFO 11896 --- [Thread-27] com.zaxxer.hikari.HikariDataSource       : HikariPool-4 - Shutdown completed.
2020-12-17 00:43:31.368  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:43:33.377  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:43:33.552  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:43:33.554  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:43:33.934  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:43:34.013  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 78ms. Found 9 JPA repository interfaces.
2020-12-17 00:43:34.134  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:43:34.144  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:43:34.385  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:43:34.386  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:43:34.421  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:43:34.510  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:43:34.552  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 990 ms
2020-12-17 00:43:34.665  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:43:34.676  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-5 - Starting...
2020-12-17 00:43:34.758  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-5 - Start completed.
2020-12-17 00:43:34.759  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:43:35.352  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:43:35.353  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:43:35.365  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:43:35.450  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:43:36.143  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:43:36.146  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:43:36.178  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:43:36.193  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:43:36.193  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:43:36.702  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:43:37.049  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 6.737 seconds (JVM running for 525.407)
2020-12-17 00:43:37.053  INFO 11896 --- [restartedMain] .ConditionEvaluationDeltaLoggingListener : Condition evaluation unchanged
2020-12-17 00:43:37.786  INFO 11896 --- [http-nio-5000-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:43:37.788  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:43:37.793  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 4 ms
2020-12-17 00:43:37.956  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:43:37.961  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:43:55.147  INFO 11896 --- [Thread-35] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:43:55.148  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:43:58.149  WARN 11896 --- [Thread-35] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:43:58.152  INFO 11896 --- [Thread-35] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:43:58.168  INFO 11896 --- [Thread-35] com.zaxxer.hikari.HikariDataSource       : HikariPool-5 - Shutdown initiated...
2020-12-17 00:43:58.182  INFO 11896 --- [Thread-35] com.zaxxer.hikari.HikariDataSource       : HikariPool-5 - Shutdown completed.
2020-12-17 00:43:59.249  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:44:01.257  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:44:01.258  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:44:01.260  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:44:01.700  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:44:01.757  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 55ms. Found 9 JPA repository interfaces.
2020-12-17 00:44:01.872  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:44:01.895  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:44:02.211  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:44:02.213  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:44:02.234  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:44:02.325  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:44:02.326  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1033 ms
2020-12-17 00:44:02.405  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:44:02.414  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-6 - Starting...
2020-12-17 00:44:02.480  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-6 - Start completed.
2020-12-17 00:44:02.482  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:44:03.111  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:44:03.112  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:44:03.146  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:44:03.222  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:44:03.683  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:44:03.685  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:44:03.717  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:44:03.719  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:44:03.742  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:44:04.421  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:44:04.971  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 6.665 seconds (JVM running for 553.33)
2020-12-17 00:44:04.974  INFO 11896 --- [restartedMain] .ConditionEvaluationDeltaLoggingListener : Condition evaluation unchanged
2020-12-17 00:44:05.703  INFO 11896 --- [http-nio-5000-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:44:05.704  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:44:05.708  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 3 ms
2020-12-17 00:44:05.862  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:44:05.870  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:44:26.560 ERROR 11896 --- [threadPoolTaskExecutor-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void by.testbot.services.async.BroadcastServiceAsync.runAsync()

org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:448) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:574) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:361) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at by.testbot.services.PostponeMessageService$$EnhancerBySpringCGLIB$$48612ad8.getAllWithoutLast(<generated>) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.idleMessagesAsync(BroadcastServiceAsync.java:57) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.runAsync(BroadcastServiceAsync.java:49) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync$$FastClassBySpringCGLIB$$1ab3ef16.invoke(<generated>) ~[classes/:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:771) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:509) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:588) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:156) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:466) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:391) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	... 21 common frames omitted

2020-12-17 00:44:55.153 ERROR 11896 --- [threadPoolTaskExecutor-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void by.testbot.services.async.BroadcastServiceAsync.runAsync()

org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:448) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:574) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:361) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at by.testbot.services.PostponeMessageService$$EnhancerBySpringCGLIB$$48612ad8.getAllWithoutLast(<generated>) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.idleMessagesAsync(BroadcastServiceAsync.java:57) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.runAsync(BroadcastServiceAsync.java:49) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync$$FastClassBySpringCGLIB$$1ab3ef16.invoke(<generated>) ~[classes/:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:771) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:509) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:588) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:156) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:466) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:391) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	... 21 common frames omitted

2020-12-17 00:45:52.740  INFO 11896 --- [Thread-43] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:45:52.740  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:45:55.742  WARN 11896 --- [Thread-43] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:45:55.744  INFO 11896 --- [Thread-43] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:45:55.759  INFO 11896 --- [Thread-43] com.zaxxer.hikari.HikariDataSource       : HikariPool-6 - Shutdown initiated...
2020-12-17 00:45:55.774  INFO 11896 --- [Thread-43] com.zaxxer.hikari.HikariDataSource       : HikariPool-6 - Shutdown completed.
2020-12-17 00:45:56.692  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:45:58.699  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:45:58.700  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:45:58.701  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:45:59.124  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:45:59.198  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 72ms. Found 9 JPA repository interfaces.
2020-12-17 00:45:59.439  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:45:59.452  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:45:59.713  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:45:59.714  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:45:59.715  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:45:59.849  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:45:59.853  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 1121 ms
2020-12-17 00:46:00.095  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:46:00.162  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-7 - Starting...
2020-12-17 00:46:00.209  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-7 - Start completed.
2020-12-17 00:46:00.249  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:46:01.102  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:46:01.103  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:46:01.160  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:46:01.359  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:46:01.917  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:46:01.918  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:46:01.929  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:46:01.943  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:46:01.944  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:46:02.490  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:46:03.127  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 7.218 seconds (JVM running for 671.485)
2020-12-17 00:46:03.129  INFO 11896 --- [restartedMain] .ConditionEvaluationDeltaLoggingListener : Condition evaluation unchanged
2020-12-17 00:46:03.879  INFO 11896 --- [http-nio-5000-exec-2] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:46:03.881  INFO 11896 --- [http-nio-5000-exec-2] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:46:03.888  INFO 11896 --- [http-nio-5000-exec-2] o.s.web.servlet.DispatcherServlet        : Completed initialization in 7 ms
2020-12-17 00:46:04.055  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:46:04.060  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:46:52.745 ERROR 11896 --- [threadPoolTaskExecutor-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void by.testbot.services.async.BroadcastServiceAsync.runAsync()

org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:448) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:574) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:361) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at by.testbot.services.PostponeMessageService$$EnhancerBySpringCGLIB$$48612ad8.getAllWithoutLast(<generated>) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.idleMessagesAsync(BroadcastServiceAsync.java:57) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.runAsync(BroadcastServiceAsync.java:49) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync$$FastClassBySpringCGLIB$$1ab3ef16.invoke(<generated>) ~[classes/:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:771) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:509) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:588) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:156) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:466) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:391) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	... 21 common frames omitted

2020-12-17 00:51:46.432  INFO 11896 --- [http-nio-5000-exec-3] by.testbot.services.viber.ViberService   : Received MessageCallback from user: gbcD9ezHUeQkbrYUwyU3Bw==, message type: TEXT
2020-12-17 00:51:47.345  INFO 11896 --- [http-nio-5000-exec-3] by.testbot.services.viber.ViberService   : File message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:51:47.508  INFO 11896 --- [http-nio-5000-exec-3] by.testbot.services.viber.ViberService   : Text message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:52:58.991  INFO 11896 --- [Thread-51] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:52:58.992  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:53:01.993  WARN 11896 --- [Thread-51] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:53:01.995  INFO 11896 --- [Thread-51] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:53:01.997  INFO 11896 --- [Thread-51] com.zaxxer.hikari.HikariDataSource       : HikariPool-7 - Shutdown initiated...
2020-12-17 00:53:02.007  INFO 11896 --- [Thread-51] com.zaxxer.hikari.HikariDataSource       : HikariPool-7 - Shutdown completed.
2020-12-17 00:53:02.862  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:53:04.872  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:53:04.873  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:53:04.876  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:53:05.241  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:53:05.304  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 63ms. Found 9 JPA repository interfaces.
2020-12-17 00:53:05.407  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:53:05.418  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:53:05.600  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:53:05.601  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:53:05.602  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:53:05.660  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:53:05.661  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 760 ms
2020-12-17 00:53:05.744  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:53:05.755  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-8 - Starting...
2020-12-17 00:53:05.789  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-8 - Start completed.
2020-12-17 00:53:05.798  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:53:06.357  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:53:06.358  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:53:06.369  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:53:06.445  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:53:06.796  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:53:06.797  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:53:06.804  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:53:06.813  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:53:06.814  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:53:07.292  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:53:07.609  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 5.461 seconds (JVM running for 1095.967)
2020-12-17 00:53:07.611  INFO 11896 --- [restartedMain] .ConditionEvaluationDeltaLoggingListener : Condition evaluation unchanged
2020-12-17 00:53:08.374  INFO 11896 --- [http-nio-5000-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:53:08.376  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:53:08.380  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 4 ms
2020-12-17 00:53:08.534  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:53:08.537  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:53:09.901  INFO 11896 --- [http-nio-5000-exec-3] by.testbot.services.viber.ViberService   : Received MessageCallback from user: gbcD9ezHUeQkbrYUwyU3Bw==, message type: TEXT
2020-12-17 00:53:10.282  INFO 11896 --- [http-nio-5000-exec-3] by.testbot.services.viber.ViberService   : File message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:53:10.458  INFO 11896 --- [http-nio-5000-exec-3] by.testbot.services.viber.ViberService   : Text message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:53:27.106  INFO 11896 --- [Thread-59] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:53:27.107  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:53:30.108  WARN 11896 --- [Thread-59] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:53:30.110  INFO 11896 --- [Thread-59] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:53:30.113  INFO 11896 --- [Thread-59] com.zaxxer.hikari.HikariDataSource       : HikariPool-8 - Shutdown initiated...
2020-12-17 00:53:30.140  INFO 11896 --- [Thread-59] com.zaxxer.hikari.HikariDataSource       : HikariPool-8 - Shutdown completed.
2020-12-17 00:53:31.299  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:53:33.319  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:53:33.320  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:53:33.321  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:53:33.622  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:53:33.677  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 55ms. Found 9 JPA repository interfaces.
2020-12-17 00:53:33.781  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:53:33.791  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:53:33.980  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:53:33.981  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:53:33.982  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:53:34.059  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:53:34.060  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 724 ms
2020-12-17 00:53:34.141  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:53:34.151  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-9 - Starting...
2020-12-17 00:53:34.184  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-9 - Start completed.
2020-12-17 00:53:34.184  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:53:34.726  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:53:34.727  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:53:34.735  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:53:34.808  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:53:35.133  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:53:35.133  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:53:35.140  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:53:35.150  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:53:35.151  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:53:35.636  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:53:35.969  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 5.644 seconds (JVM running for 1124.327)
2020-12-17 00:53:35.972  INFO 11896 --- [restartedMain] .ConditionEvaluationDeltaLoggingListener : Condition evaluation unchanged
2020-12-17 00:53:36.687  INFO 11896 --- [http-nio-5000-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:53:36.688  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:53:36.697  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 8 ms
2020-12-17 00:53:36.848  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:53:36.852  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:53:58.998 ERROR 11896 --- [threadPoolTaskExecutor-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void by.testbot.services.async.BroadcastServiceAsync.runAsync()

org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:448) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:574) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:361) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at by.testbot.services.PostponeMessageService$$EnhancerBySpringCGLIB$$48612ad8.getAllWithoutLast(<generated>) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.idleMessagesAsync(BroadcastServiceAsync.java:57) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.runAsync(BroadcastServiceAsync.java:49) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync$$FastClassBySpringCGLIB$$1ab3ef16.invoke(<generated>) ~[classes/:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:771) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:509) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:588) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:156) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:466) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:391) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	... 21 common frames omitted

2020-12-17 00:54:27.132 ERROR 11896 --- [threadPoolTaskExecutor-1] .a.i.SimpleAsyncUncaughtExceptionHandler : Unexpected exception occurred invoking async method: public void by.testbot.services.async.BroadcastServiceAsync.runAsync()

org.springframework.transaction.CannotCreateTransactionException: Could not open JPA EntityManager for transaction; nested exception is java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:448) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.startTransaction(AbstractPlatformTransactionManager.java:400) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.getTransaction(AbstractPlatformTransactionManager.java:373) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.createTransactionIfNecessary(TransactionAspectSupport.java:574) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:361) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:118) ~[spring-tx-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:691) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at by.testbot.services.PostponeMessageService$$EnhancerBySpringCGLIB$$48612ad8.getAllWithoutLast(<generated>) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.idleMessagesAsync(BroadcastServiceAsync.java:57) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync.runAsync(BroadcastServiceAsync.java:49) ~[classes/:na]
	at by.testbot.services.async.BroadcastServiceAsync$$FastClassBySpringCGLIB$$1ab3ef16.invoke(<generated>) ~[classes/:na]
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218) ~[spring-core-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:771) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:749) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalStateException: EntityManagerFactory is closed
	at org.hibernate.internal.SessionFactoryImpl.validateNotClosed(SessionFactoryImpl.java:509) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:588) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.hibernate.internal.SessionFactoryImpl.createEntityManager(SessionFactoryImpl.java:156) ~[hibernate-core-5.4.22.Final.jar:5.4.22.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.createEntityManagerForTransaction(JpaTransactionManager.java:466) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	at org.springframework.orm.jpa.JpaTransactionManager.doBegin(JpaTransactionManager.java:391) ~[spring-orm-5.2.10.RELEASE.jar:5.2.10.RELEASE]
	... 21 common frames omitted

2020-12-17 00:56:06.639  INFO 11896 --- [http-nio-5000-exec-3] by.testbot.services.viber.ViberService   : Received MessageCallback from user: gbcD9ezHUeQkbrYUwyU3Bw==, message type: TEXT
2020-12-17 00:56:07.483  INFO 11896 --- [http-nio-5000-exec-3] by.testbot.services.viber.ViberService   : File message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:56:07.620  INFO 11896 --- [http-nio-5000-exec-3] by.testbot.services.viber.ViberService   : Text message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:56:32.841  INFO 11896 --- [Thread-67] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:56:32.842  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:56:35.843  WARN 11896 --- [Thread-67] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:56:35.846  INFO 11896 --- [Thread-67] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:56:35.858  INFO 11896 --- [Thread-67] com.zaxxer.hikari.HikariDataSource       : HikariPool-9 - Shutdown initiated...
2020-12-17 00:56:35.869  INFO 11896 --- [Thread-67] com.zaxxer.hikari.HikariDataSource       : HikariPool-9 - Shutdown completed.
2020-12-17 00:56:36.712  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Fetching config from server at : http://localhost:8888
2020-12-17 00:56:38.720  INFO 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Connect Timeout Exception on Url - http://localhost:8888. Will be trying the next url if available
2020-12-17 00:56:38.721  WARN 11896 --- [restartedMain] c.c.c.ConfigServicePropertySourceLocator : Could not locate PropertySource: I/O error on GET request for "http://localhost:8888/application/default": Connection refused: connect; nested exception is java.net.ConnectException: Connection refused: connect
2020-12-17 00:56:38.722  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : No active profile set, falling back to default profiles: default
2020-12-17 00:56:39.011  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Bootstrapping Spring Data JPA repositories in DEFAULT mode.
2020-12-17 00:56:39.065  INFO 11896 --- [restartedMain] .s.d.r.c.RepositoryConfigurationDelegate : Finished Spring Data repository scanning in 52ms. Found 9 JPA repository interfaces.
2020-12-17 00:56:39.159  INFO 11896 --- [restartedMain] o.s.cloud.context.scope.GenericScope     : BeanFactory id=4364c041-4530-3f37-b030-d2baa5ef1efb
2020-12-17 00:56:39.169  INFO 11896 --- [restartedMain] trationDelegate$BeanPostProcessorChecker : Bean 'by.testbot.proxy.ViberProxy' of type [org.springframework.cloud.openfeign.FeignClientFactoryBean] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2020-12-17 00:56:39.307  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 5000 (http)
2020-12-17 00:56:39.308  INFO 11896 --- [restartedMain] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-12-17 00:56:39.309  INFO 11896 --- [restartedMain] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.39]
2020-12-17 00:56:39.373  INFO 11896 --- [restartedMain] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-12-17 00:56:39.373  INFO 11896 --- [restartedMain] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 634 ms
2020-12-17 00:56:39.447  INFO 11896 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
2020-12-17 00:56:39.457  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-10 - Starting...
2020-12-17 00:56:39.484  INFO 11896 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-10 - Start completed.
2020-12-17 00:56:39.486  INFO 11896 --- [restartedMain] org.hibernate.dialect.Dialect            : HHH000400: Using dialect: org.hibernate.dialect.MySQL5InnoDBDialect
2020-12-17 00:56:40.006  INFO 11896 --- [restartedMain] o.h.e.t.j.p.i.JtaPlatformInitiator       : HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
2020-12-17 00:56:40.007  INFO 11896 --- [restartedMain] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:56:40.017  INFO 11896 --- [restartedMain] o.s.b.d.a.OptionalLiveReloadServer       : LiveReload server is running on port 35729
2020-12-17 00:56:40.084  WARN 11896 --- [restartedMain] JpaBaseConfiguration$JpaWebConfiguration : spring.jpa.open-in-view is enabled by default. Therefore, database queries may be performed during view rendering. Explicitly configure spring.jpa.open-in-view to disable this warning
2020-12-17 00:56:40.649  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService
2020-12-17 00:56:40.650  INFO 11896 --- [restartedMain] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:56:40.657  WARN 11896 --- [restartedMain] o.s.c.n.a.ArchaiusAutoConfiguration      : No spring.application.name found, defaulting to 'application'
2020-12-17 00:56:40.662  WARN 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : No URLs will be polled as dynamic configuration sources.
2020-12-17 00:56:40.663  INFO 11896 --- [restartedMain] c.n.c.sources.URLConfigurationSource     : To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
2020-12-17 00:56:41.123  INFO 11896 --- [restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 5000 (http) with context path ''
2020-12-17 00:56:41.432  INFO 11896 --- [restartedMain] by.testbot.TestbotApplication            : Started TestbotApplication in 5.433 seconds (JVM running for 1309.79)
2020-12-17 00:56:41.435  INFO 11896 --- [restartedMain] .ConditionEvaluationDeltaLoggingListener : Condition evaluation unchanged
2020-12-17 00:56:42.259  INFO 11896 --- [http-nio-5000-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-12-17 00:56:42.261  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-12-17 00:56:42.266  INFO 11896 --- [http-nio-5000-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 5 ms
2020-12-17 00:56:42.409  INFO 11896 --- [restartedMain] by.testbot.services.viber.ViberService   : Webhook setted with response: ok, events: [SUBSCRIBED, UNSUBSCRIBED, CONVERSATION_STARTED, FAILED, MESSAGE]
2020-12-17 00:56:42.414  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service started
2020-12-17 00:56:43.597  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : Received MessageCallback from user: gbcD9ezHUeQkbrYUwyU3Bw==, message type: TEXT
2020-12-17 00:56:43.881  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : File message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:56:44.039  INFO 11896 --- [http-nio-5000-exec-2] by.testbot.services.viber.ViberService   : Text message sended to user: gbcD9ezHUeQkbrYUwyU3Bw==
2020-12-17 00:57:14.213  INFO 11896 --- [Thread-75] o.s.s.concurrent.ThreadPoolTaskExecutor  : Shutting down ExecutorService 'threadPoolTaskExecutor'
2020-12-17 00:57:14.213  INFO 11896 --- [threadPoolTaskExecutor-1] b.t.s.async.BroadcastServiceAsync        : Postpone message service stopped: sleep interrupted
2020-12-17 00:57:17.214  WARN 11896 --- [Thread-75] o.s.s.concurrent.ThreadPoolTaskExecutor  : Timed out while waiting for executor 'threadPoolTaskExecutor' to terminate
2020-12-17 00:57:17.226  INFO 11896 --- [Thread-75] j.LocalContainerEntityManagerFactoryBean : Closing JPA EntityManagerFactory for persistence unit 'default'
2020-12-17 00:57:17.244  INFO 11896 --- [Thread-75] com.zaxxer.hikari.HikariDataSource       : HikariPool-10 - Shutdown initiated...
2020-12-17 00:57:17.281  INFO 11896 --- [Thread-75] com.zaxxer.hikari.HikariDataSource       : HikariPool-10 - Shutdown completed.
